content/8.x/admin/installation-guide/Upgrading.ru.adoc

```diff
@@ -0,0 +1,43 @@
+---
+Title: Обновление системы
+Weight: 30
+---
+
+:author: likhobory
+:email: likhobory@mail.ru
+
+
+:toc:
+:toc-title: Оглавление
+:toclevels: 1
+
+:experimental:
+
+:imagesdir: /images/ru/admin/Upgrading
+
+ifdef::env-github[:imagesdir: ./../../../../master/static/images/ru/admin/Upgrading]
+
+:btn: btn:
+
+ifdef::env-github[:btn:]  
+
+{{% notice note %}}
```

Comment:
Better to have info or note?

Reply:
Depends on the context, but in this case I considered this message to be quite important, I mark less important messages as info. IMHO.